### DISCUSSION TITLE
Planes never drop off of the map in VRS

### DISCUSSION THREAD

**Discussion by jaymot123** [szpajder/dumphfdl#22] [Q&A] [answered]
I just got an active loop antenna and along with my SDRPlay RSP1A I'm _finally_ able to receive HFDL messages. I piped the output to 127.0.0.1:20005 so I can watch the planes in VRS but they never drop off of the map even with position ages of almost an hour.  The normal time-out in VRS for ADS-B is 60 seconds, but there must be a setting somewhere for the HFDL receiver that I'm not finding? My idle time-out is set to 1800 seconds which is 30 minutes. This is with dumphfdl 1.2.0 and VRS 3.0.0.6 Mono version for Linux.

Edit: upon watching VRS for a while I see that they do drop off of the map after an hour. Is that from the idle time-out setting? Or is the one hour time-out coming from dumphfdl?

**Comment by dlpwx** [comment, marked as answer]
In VRS Admin, Settings, General list of options JAERO tracking timeout options that default to 60 minutes (map) and 120 minutes (list). I changed them to both be 60 minutes and pretty sure that worked to keep the map and list showing the same number of planes. With an hour, you can watch the tracks lines grow pretty long on good days, say when planes fly across the Pacific ocean with sometimes long gaps between positions. 

With your RSP1A sample rate around let's say 6 covering multiple frequency groups, you might be able to occasionally track a long 10 or 12 hr flight even as it changes frequencies over the course of the day. Depends on the ionosphere and frequencies lining up just right. That's why I chose to stick with one-hour tracking timeouts. It makes the map a little more interesting. Also recommend setting up multitail per the readme instructions to watch real time and possibly tuning.

**Reply by jaymot123** [reply]
"With your RSP1A sample rate around let's say 6 covering multiple frequency groups" Wait, I can do that? Like monitor frequencies in the 5, 6, 8 and 11MHz bands at the same time? I've just been doing one band at a time. I'm basically starting over from scratch. Not only is today the first day I've received messages, I haven't had much time to even use the RSP1A and figure out its capabilities. I lost power for a month and a half (typhoon Odette) and just got it back last Thursday.

**Reply by dlpwx** [reply]
Sure you can. Though my tests at 10 sample rate trigger occasional read stream errors in spite of robust processor and PCIE USB card. 6 has no problem though. Per readme, just keep the frequency range within maybe 80% of the sample rate range. e.g. 
--soapysdr driver=sdrplay --sample-rate 6000000 8834 8843 8885 8886 8894 8912 8921 8925 8927 8936 8939 8942 8948 8957 8977 10027 10060 10063 10066 10075 10081 10084 10087 10093 11184 11306 11312 11318 11321 11327 11348 11354 11384 11387 13264 13270 13276 13303 13312 13315 13321 13342 13351 13354"

**Reply by jaymot123** [reply]
Thank you!

**Reply by jaymot123** [reply]
FYI the K-180WLA active loop antenna I bought came with 10 meters of RG174 TNC to SMA coax to run between the amp at the bottom of the loop and the bias-T power supply. I just got a TNC to SMA adapter and swapped out that RG174 for 15 meters of RG58 that I had, and it's like night and day! I'm seeing twice as many planes in VRS and many more showing trails indicating I'm getting multiple messages for them.  Just using halfway-decent coax with better shielding and thicker dielectric made a huge boost in performance. 

It's just early afternoon here so only the 17 and 21MHz bands are really usable right now, but I'm really looking forward to this evening when the lower bands are receivable to see if I can get anything from ground stations other than Guam, S. Korea and New Zealand and see planes on transoceanic flights, over Europe, the Americas etc.
